### PR TITLE
fix: simple add attr value for jquery error

### PR DESCRIPTION
**Pull request by Bound3R**
Is for this reason the button stay blocked... :lock:

### CODE REVIEW

File: lms/templates/login.html [hunk 1]
```diff
@@ -100,7 +100,7 @@
         $submitButton.
           removeClass('is-disabled').
           attr('aria-disabled', false).
-          removeProp('disabled').
+          prop('disabled', false).
```
Comment (felipemontoya) [inline lms/templates/login.html]: Do we need to change this at the register template also? here and in the theme?

Comment (felipemontoya) [inline lms/templates/login.html]: I just saw the theme PR. :+1: 
I'd say we should also fix that here at the `register.html` template

Comment (Bound3R) [inline lms/templates/login.html]: sure, this is for login submit button and the other in theme is of register submit button. :facepunch:

Comment (felipemontoya) [inline lms/templates/login.html]: That is what I mean, lets change it also here, and not only on the theme

**Pull request merged**